server/routes/induction/create/reasonsNotToGetWorkCreateController.test.ts

```diff
@@ -0,0 +1,202 @@
+      }
+      req.session.reasonsNotToGetWorkForm = expectedReasonsNotToGetWorkForm
+
+      const expectedView = {
+        prisonerSummary,
+        form: expectedReasonsNotToGetWorkForm,
+        backLinkUrl: '/prisoners/A1234BC/create-induction/hoping-to-work-on-release',
+        backLinkAriaText: `Back to Is Jimmy Lightfingers hoping to get work when they're released?`,
+        errors: noErrors,
+      }
+
+      // When
+      await controller.getReasonsNotToGetWorkView(req, res, next)
+
+      // Then
+      expect(res.render).toHaveBeenCalledWith('pages/induction/reasonsNotToGetWork/index', expectedView)
+      expect(req.session.hopingToWorkOnReleaseForm).toBeUndefined()
+      expect(req.session.inductionDto).toEqual(inductionDto)
+    })
+  })
+
+  describe('submitReasonsNotToGetWorkForm', () => {
+    it('should redisplay page given form is submitted with validation errors', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+      const inductionDto = { prisonNumber } as InductionDto
+      req.session.inductionDto = inductionDto
+      req.body = {}
+
+      const expectedErrors = [
+        {
+          href: '#reasonsNotToGetWork',
+          text: `Select what could stop Jimmy Lightfingers getting work on release, or select 'Not sure'`,
+        },
+      ]
+
+      // When
+      await controller.submitReasonsNotToGetWorkForm(req, res, next)
+
+      // Then
+      expect(res.redirect).toHaveBeenCalledWith('/prisoners/A1234BC/create-induction/reasons-not-to-get-work')
+      expect(req.flash).toHaveBeenCalledWith('errors', expectedErrors)
+      expect(req.session.hopingToWorkOnReleaseForm).toEqual(undefined)
+      expect(req.session.inductionDto).toEqual(inductionDto)
+    })
+
+    it('should update induction DTO and redirect to want to add qualifcations page', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+      const inductionDto = { prisonNumber } as InductionDto
+      req.session.inductionDto = inductionDto
+
+      req.body = {
+        reasonsNotToGetWork: ['FULL_TIME_CARER', 'LACKS_CONFIDENCE_OR_MOTIVATION'],
+      }
+
+      const expectedInduction = {
+        prisonNumber,
+        workOnRelease: {
+          notHopingToWorkReasons: ['FULL_TIME_CARER', 'LACKS_CONFIDENCE_OR_MOTIVATION'],
+        },
+      } as InductionDto
+
+      // When
+      await controller.submitReasonsNotToGetWorkForm(req, res, next)
+
+      // Then
+      expect(res.redirect).toHaveBeenCalledWith('/prisoners/A1234BC/create-induction/want-to-add-qualifications')
+      expect(req.flash).toHaveBeenCalledTimes(0)
+      expect(req.session.hopingToWorkOnReleaseForm).toEqual(undefined)
+      expect(req.session.inductionDto).toEqual(expectedInduction)
+    })
+
+    it('should update induction DTO and redirect to qualifcations page', async () => {
```

Comment:
Ah, have you copied/based this on the update journey?
I don't think this is a valid use case for the Create journey, as there will never be existing qualifications on a new induction at this point (because we've not asked the user about them yet 😁 )

Reply:
Looking at the user flow I think this wrong anyway because it seems for the create journey there is another page in between this page and the qualifications page.

That said, wouldn't the logic be needed if the user returns to the page from check answers? That was my thinking anyway.

Reply:
100% the logic will need updating when we do Check Your Answers, but we'll come back to do that once we've got the straight forward journey working first 😁 
It's complicated enough without us adding in complexity we don't need yet ! 🤣